kakarot_scripts/ef_tests/fetch.py

```diff
@@ -32,20 +32,37 @@ def generate_tests():
         f"{os.path.basename(root)}_{name}": content
         for (root, _, files) in os.walk(EF_TESTS_DIR)
         for file in files
-        if file.endswith(".json") and "BlockchainTests/GeneralStateTests" in root
+        if file.endswith(".json")
+        and "BlockchainTests/GeneralStateTests" in root
+        and "Pyspecs" not in root
         for name, content in json.loads((Path(root) / file).read_text()).items()
         if content.get("network") == DEFAULT_NETWORK
     }
 
+    # Add tests from BlockchainTests/GeneralStateTests/Pyspecs
+    test_cases.update(
+        {
+            f"{name.split('::')[-1]}": content
```

Comment:
the name convention changes then?

Reply:
Pyspec tests have different names, e.g. `src/GeneralStateTestsFiller/Pyspecs/berlin/eip2930_access_list/test_acl.py::test_access_list[fork_Berlin-blockchain_test]` instead of the regular `access_list_d0g0v0` for non-python based tests

For them